tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) 2023-2024, Nubificus LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package urunce2etesting
+
+import (
+	"testing"
+)
+
+type testMethod func(tool testTool) error
```

Comment:
I have moved the struct and the `testMethod` declaration to `tests_skeleton.go` file. I opted to keep `common.go` file free from test related code, in case we decide to make it a package later.